CodeGeneration/Sources/generate-swiftsyntax/templates/basicformat/BasicFormatFile.swift

```diff
@@ -141,11 +155,20 @@ let basicFormatFile = SourceFile {
         switch (token.tokenKind, token.nextToken(viewMode: .sourceAccurate)?.tokenKind) {
         case (.asKeyword, .exclamationMark),
              (.asKeyword, .postfixQuestionMark),
+             (.exclamationMark, .period),
              (.initKeyword, .leftParen),
              (.initKeyword, .postfixQuestionMark),
+             (.leftAngle, .identifier(_)),
+             (.rightAngle, .leftParen),
+             (.rightAngle, .postfixQuestionMark),
+             (.postfixQuestionMark, .leftParen),
+             (.postfixQuestionMark, .rightAngle),
+             (.postfixQuestionMark, .rightParen),
              (.tryKeyword, .exclamationMark),
              (.tryKeyword, .postfixQuestionMark):
           return false
+        case (.spacedBinaryOperator(let `operator`), .comma):
+          return `operator` != "*"
```

Comment:
https://github.com/apple/swift-syntax/pull/1164#discussion_r1060621893

I only found this case. Will not affect more than necessary

Reply:
I would prefer to decide on whether the operator is `*` or not.

Either this rule works in all situations in which operators occur, in which case it should also apply for e.g. `+`. If it breaks other operators in expression positions, it will also be broken for `*`.

Reply:
So you suggest we do it for all operators?

Reply:
Yes, that's what I'm suggesting. Because otherwise it will be surprising why `*` is formatted differently than `+`.

Reply:
Fixed. We do now do the same for alle spaced binary operators